extensions/ql-vscode/src/data-extensions-editor/model-details/model-details-panel.ts

```diff
@@ -24,4 +28,11 @@ export class ModelDetailsPanel extends DisposableObject {
   ): void {
     this.dataProvider.setState(externalApiUsages, databaseItem);
   }
+
+  public async revealItem(usage: Usage): Promise<void> {
+    const canonicalUsage = this.dataProvider.resolveCanonicalUsage(usage);
```

Comment:
The public interface of the model details panel and data provider are becoming a little larger and less well defined. I don't know if it's a problem yet but probably worth keeping an eye on and thinking about if we could make things clearer by moving methods around or changing which object is the public interface to the rest of the code.